engine/src/main/java/org/hibernate/search/indexes/impl/IndexManagerHolder.java

```diff
@@ -211,6 +213,14 @@ public IndexManager getIndexManager(String targetIndexName) {
 		return indexManagersRegistry.get( targetIndexName );
 	}
 
+	public BackendQueueProcessor getBackendQueueProcessor(String indexName) {
```

Comment:
I also considered a class `Backend` comprising the index manager and the queue processor of a given index but I decided against it as that would have been much more invasive.